### PR TITLE
Added Listener arns in outputs #minor

### PR DESCRIPTION
**Pull request by mayank0202**

### CODE REVIEW

**Review by bcrysler — APPROVED** [review]

**Review by tsaucier-sf — APPROVED** [review]

File: outputs.tf [hunk 1]
```diff
@@ -44,9 +44,14 @@ output "alb_https_listener_arn" {
   description = "HTTPs listener ARN for downstream services to use"
```
Comment (tsaucier-sf) [inline outputs.tf:44]: Make HTTPS

File: outputs.tf [hunk 2]
```diff
@@ -44,9 +44,14 @@ output "alb_https_listener_arn" {
...
 
+output "alb_http_listener_arn" {
+  value       = aws_lb_listener.http.arn
+  description = "HTTP listener ARN for downstream services to use"
+}
+
 output "alb_security_group_id" {
   value       = module.alb_sg.id
-  description = "HTTPs listener ARN for downstream services to use"
+  description = "Alb Security Group"
```
Comment (tsaucier-sf) [inline outputs.tf:54]: Change to ALB

File: locals.tf [hunk 3]
```diff
@@ -29,6 +29,19 @@ locals {
       description = "ALB Health Check FQDN."
       type        = "String"
     },
+    ## listeners
+    {
+      name        = "/${var.namespace}/${var.environment}/alb/${module.alb.alb_name}/http_listener/arn"
```
Comment (tsaucier-sf) [inline locals.tf:34]: Hyphenate http_listener --> http-listener

File: locals.tf [hunk 4]
```diff
@@ -29,6 +29,19 @@ locals {
...
+      value       = aws_lb_listener.http.arn
+      description = "ARN of the HTTP listener"
+      type        = "String"
+    },
+    {
+      name        = "/${var.namespace}/${var.environment}/alb/${module.alb.alb_name}/https_listener/arn"
```
Comment (tsaucier-sf) [inline locals.tf:40]: Hyphenate https_listener --> https-listener

**Review by tsaucier-sf — CHANGES_REQUESTED** [review]

**Review by tsaucier-sf — APPROVED** [review]

**Pull request merged**